web/react/stores/browser_store.jsx

```diff
@@ -12,43 +12,57 @@ function getPrefix() {
 // Also change model/utils.go ETAG_ROOT_VERSION
 var BROWSER_STORE_VERSION = '.5';
 
-module.exports = {
-    initialized: false,
+class BrowserStoreClass {
+    constructor() {
+        this.initialize = this.initialize.bind(this);
+        this.getItem = this.getItem.bind(this);
+        this.setItem = this.setItem.bind(this);
+        this.removeItem = this.removeItem.bind(this);
+        this.setGlobalItem = this.setGlobalItem.bind(this);
+        this.getGlobalItem = this.getGlobalItem.bind(this);
+        this.removeGlobalItem = this.removeGlobalItem.bind(this);
+        this.clear = this.clear.bind(this);
+        this.actionOnItemsWithPrefix = this.actionOnItemsWithPrefix.bind(this);
+        this.isLocalStorageSupported = this.isLocalStorageSupported.bind(this);
+
+        this.initialized = false;
```

Comment:
Yah, that makes sense, I moved the initialize into the constructor so as long as only one object is created that shouldn't be a problem.